server/chat-plugins/trivia.ts

```diff
@@ -71,24 +68,45 @@ const INTERMISSION_INTERVAL = 20 * 1000;
 const MAX_QUESTION_LENGTH = 252;
 const MAX_ANSWER_LENGTH = 32;
 
-/**
- * @typedef {{question: string, category: string, answers: string[], user?: string, type?: string}} TriviaQuestion
- * @typedef {[number, number, number]} TriviaRank
- * @typedef {TriviaQuestion[]} TriviaQuestions
- * @typedef {{[k: string]: TriviaRank}} TriviaLeaderboard
- * @typedef {TriviaRank[][]} TriviaLadder
- * @typedef {{questions?: TriviaQuestions, submissions?: TriviaQuestions, leaderboard?: TriviaLeaderboard, ladder?: TriviaLadder}} TriviaData
- */
+interface TriviaQuestion {
+	question: string;
+	category: string;
+	answers: string[];
+	user?: string;
+	type?: string;
+}
+
+// Should be [number, number, number]
+type TriviaRank = [number, number, number];
```

Comment:
...is that comment still necessary?

```suggestion
type TriviaRank = [number, number, number];
```